SPECS/libdivsufsort/libdivsufsort.spec

```diff
@@ -0,0 +1,42 @@
+Summary:        A software library that implements a lightweight suffix array construction algorithm.
+Name:           libdivsufsort
+Version:        2.0.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Libraries
+URL:            https://github.com/y-256/libdivsufsort
+# Source0:      https://github.com/y-256/libdivsufsort/archive/refs/tags/%%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  make
+
+%description
+A software library that implements a lightweight suffix array construction algorithm.
+
+%prep
+%autosetup
+
+%build
+mkdir build
+cd build
+%cmake -DCMAKE_BUILD_TYPE="Release" -DCMAKE_INSTALL_PREFIX="/usr" ..
+make %{?_smp_mflags}
+
+%install
+cd build
+%make_install
+
+%clean
+rm -rf %{buildroot}/*
+
+%files
+%{_includedir}/divsufsort.h
+%{_libdir}/libdivsufsort.so*
```

Comment:
Could you also add the major version number (`%{_libdir}/libdivsufsort.so.X*`) to the list? It will make spotting updates easier.